pkg/wallet/legacy_api.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package wallet
+
+import "io"
+
+// LegacySecretBox provides methods for wallet to perform secret box sealing and unsealing
+type LegacySecretBox interface {
```

Comment:
Why is there a "Legacy" interface in the **generic** wallet package?

Reply:
What interface :^)

Replaced with a Box struct that implements these methods, that the wallet instantiates.